packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/editor/inputs/DatabaseComboBox.kt

```diff
@@ -78,13 +80,9 @@ class DatabaseComboBox(
         var isFirstInit = true
         coroutineScope.launch {
             project.getToolbarModel().toolbarState.collect { state ->
-                withoutSelectionChangedListener {
-                    loadingDatabases = state.databasesLoadingForSelectedDataSource
-                    if (isFirstInit || state.databases != databases) {
-                        isFirstInit = false
-                        populateComboBoxWithDatabases(state.databases)
-                    }
-                    selectDatabaseAndNotify(state.selectedDatabase)
+                SwingUtilities.invokeLater {
```

Comment:
Same comment as above, is there an specific reason to use SwingUtilities instead of Application invokeLater?